Sources/Helpers/PrepareForSkeletonProtocol.swift

```diff
@@ -12,22 +12,50 @@ protocol PrepareForSkeleton {
     func prepareViewForSkeleton() 
 }
 
-extension UILabel: PrepareForSkeleton {
-    func prepareViewForSkeleton() {
-        text = " "
-        resignFirstResponder()
+extension UIView: PrepareForSkeleton {
+    @objc func prepareViewForSkeleton() {
+        _ = startTransitionIfAvailable()
+    }
+    fileprivate func startTransitionIfAvailable() -> Bool {
+        if let transitionType = currentSkeletonConfig?.transition {
+            switch transitionType {
+            case .none:
```

Comment:
I see what do you want to do with fade with duration 0, but I think we shouldn't do this, we can do a fade animation with 0 secs, doesn't make sense, but the developer can do it, it's not a problem for the library. What do you think?